### PR TITLE
Accessibility audit changes

### PR DESCRIPTION
**Pull request by jonathanglassman**
### Context

The content within the tech docs has been audited to ensure it complies with accessibility regulation before the 22 September 2020 deadline.

### Changes proposed in this pull request

- Removed inaccessible language like whitelist and blacklist and replaced with "allow list" and "block list"
- Split up long paragraphs
- Ensured all diagrams and images has alt text or content in surrounding text explaining them

### Guidance to review

Check that content and structure makes sense.

### CODE REVIEW

**Comment by jonathanglassman** [comment]
Jira ticket: https://payments-platform.atlassian.net/browse/PP-6785

File: source/payment_flow/index.html.md.erb [hunk 1]
```diff
@@ -58,7 +59,7 @@ When your user reaches a page in your service where they need to make a payment,
 
 You should also have a clear call to action that tells your user they're about to pay. For example, a button that says __Pay now__ or __Continue to payment__.
 
-You do not need to tell your user that they'll be taken to GOV.UK Pay’s pages to make their payment.
+You do not need to tell your user that they'll be taken to GOV.UK Pay’s pages to make their payment. The following image shows an example payment page.
 
 <%= image_tag "/images/flow-service-payment-page.png", { :alt => '' } %>
```
Comment (iqbalgds) [inline source/payment_flow/index.html.md.erb:64]: Do we need an ALT tag here?

Comment (jonathanglassman) [reply source/payment_flow/index.html.md.erb:64]: @iqbalgds If you explain the image in the main text, you should use alt=”” as this reduces auditory clutter with 
screen readers. I added the text "The following image shows an example payment page" to do this. The other images in this page aren't explicitly explained in the main text, and doing so would have meant more changes to the content than I think necessary.

Comment (iqbalgds) [reply source/payment_flow/index.html.md.erb:64]: Thanks

Comment (iqbalgds) [inline source/payment_flow/index.html.md.erb:64]: I believe we still need an alt tag for accessibility even if the text explains it.

Comment (iqbalgds) [reply source/payment_flow/index.html.md.erb]: This has been checked with content accessibility team - so happy to approve.

**Pull request merged**